parm/archive/gdas.yaml.j2

```
@@ -101,6 +101,7 @@ gdas:
                 {% endfor %}
         - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ head }}snowstat.tgz"
         - "{{ COMIN_CONF | relpath(ROTDIR) }}/{{ head }}snowanlvar.yaml"
+        - "{{ COMIN_SNOW_ANLMON | relpath(ROTDIR) }}/*output_snow.nc"
```

Could this be made more explicit?  Based on parm/gdas/stat_base_config.yaml.j2, I think this could be expanded to

```suggestion
        - "{{ COMIN_SNOW_ANLMON | relpath(ROTDIR) }}/ims_snow_{{ cycle_YMDH }}_output_snow.nc"
        - "{{ COMIN_SNOW_ANLMON | relpath(ROTDIR) }}/sfcsno_{{ cycle_YMDH }}_output_snow.nc"
        - "{{ COMIN_SNOW_ANLMON | relpath(ROTDIR) }}/snocvr_{{ cycle_YMDH }}_output_snow.nc"
```

---

Only issue (not a big one) is that a PR has to be opened that changes this file if an ob type is added/removed. I'm wondering if it is better to do a tar file when they are created and then only archive one tar file? like how the raw `*stat` files are archived?

---

I like that option better, Cory.

---

Thanks both for reviewing this PR. I will work on this. Convert to draft during edits.

---

Thank you @yuanxue2870 (and @DavidHuber-NOAA )